guide/package.json

```diff
@@ -33,9 +33,9 @@
 		"eslint": "^8.35.0",
 		"tailwindcss": "^3.4.0",
 		"typescript": "^5.3.3",
-		"wrangler": "^3.22.1"
+		"wrangler": "^3.28.2"
```

Comment:
running `pnpm dedupe` said these versions were the correct requested versions from `@remix-run/dev`